gradle.properties

```diff
@@ -13,7 +13,7 @@ cordaProductVersion = 5.0.0
 cordaApiRevision = 754
```

Comment:
Arguably, it is a breaking change, hence version needs to be bumped.

Reply:
It _shouldn't_ have been a breaking change, because that dependency no longer has any business _existing_ in `corda-api`.